datacube_alchemist/settings.py

```diff
@@ -34,12 +33,11 @@ def _convert_write_data_settings(settings):
 @attr.s(auto_attribs=True)
 class OutputSettings:
     location: str
-    dtype: np.dtype
-    nodata: int  # type depends on dtype
     write_data_settings: Optional[Mapping[str, str]] = attr.ib(
         converter=_convert_write_data_settings
     )
-    preview_image: Optional[List[str]] = None
+    nodata: Optional[int] = None
+    preview_image: Optional[Mapping[Any, Any]] = None
```

Comment:
Hey @omad what do you think of this change?

It increases flexibility, but breaks existing configs... I can't decide if it is a good idea.

Reply:
I think it's a good idea. I'm not sure what options eo-datasets3 support for preview images, but we would want to expose all of them.

We could make this backwards compatible, but, I don't think it's worth the extra code.